code/game/machinery/Sleeper.dm

```diff
@@ -305,6 +321,16 @@
 	to_chat(occupant, SPAN_WARNING("You feel a tube jammed down your throat."))
 	pump = !pump
 
+/obj/machinery/sleeper/proc/toggle_lavage()
+	if(!occupant || !beaker)
+		lavage = 0
+		return
+	lavage = !lavage
+	if (lavage)
+		to_chat(occupant, SPAN_WARNING("You feel a tube jammed down your throat."))
```

Comment:
Maybe swap `throat` to `windpipe` for clarity?

Reply:
That works!